src/shop.js

```diff
@@ -438,8 +439,11 @@ function BinartaShopjs(checkpoint, deps) {
             if (!fsm.getPaymentProvider())
                 if (shop.localStorage.getItem('binartaJSPaymentProvider'))
                     fsm.setPaymentProvider(shop.localStorage.getItem('binartaJSPaymentProvider'));
-                else
-                    fsm.setPaymentProvider('wire-transfer');
+                else {
+                    var profile = application.profile();
+                    if (profile.availablePaymentMethods && profile.availablePaymentMethods.length == 1)
+                        fsm.setPaymentProvider(profile.availablePaymentMethods[0]);
```

Comment:
That's a fair decision to make but let's make sure UI's that don't communicate validation errors related to not selecting one are rendered.